.github/.dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
```

Comment:
Do you know if it also finds requirements.txt inside demo-code when its referenced from requirements-dev.txt? (I don't know- if not, I think another entry is needed?)

Reply:
I don't know if it does or not